src/foam/core/reflow/Console.js

```diff
@@ -114,14 +113,25 @@ foam.CLASS({
       border: none;
       color: $black;
     }
+    ^title .foam-u2-TextInputCSS::placeholder {
+      color: $textDefault;
+      opacity: 1;
+    }
     ^header-actions {
       display: flex;
-      gap: 10px;
+      gap: 5px;
       align-items: center;
     }
     ^save-text {
       color: $grey700;
     }
+    ^ .foam-u2-view-OverlayActionListView {
+      color: $textDefault;
+    }
+    ^ .foam-u2-ActionView[name="back"]:not([disabled="disabled"]),
+    ^ .foam-u2-ActionView[name="forth"]:not([disabled="disabled"]) {
+      color: $textDefault;
```

Comment:
Is there some design requirement that needs you to change this? Buttons are intentionally not set to textDefault as the base colour since everything on a page being the same colour makes it way harder to parse the screen quickly. 

Hence why most buttons are grey-ish so they are not distracting from the main content of the page and then the primary button stands out (in this case, save) when you do need to look for one.

Reply:
Or alternatively in your repo you can refine button and change the css for tertiary to give you black buttons if you never use the grey ones. That way default foam gets to keep its styling as well